app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -1519,13 +1519,23 @@ protected void setTitle() {
                     setTitle(R.string.drawer_item_shared);
                     break;
                 default:
-                    setTitle(themeUtils.getDefaultDisplayNameForRootFolder(getContext()));
+                    setTitle(themeUtils.getDefaultDisplayNameForRootFolder(getContext()), isRoot());
                     break;
             }
         }
 
     }
 
+    //NMC Customization
+    //for NMC we are using defaultToolbar instead searchToolbar for which we needed customization
+    private boolean isRoot() {
```

Comment:
Before closing this PR once NC merged their PR. We still have to keep this changes as it is our customization.